test/unit/listeners/substrate-listener/commands/orders/order-paid/order-paid.command.spec.ts

```diff
@@ -0,0 +1,50 @@
+import { CommandBus, CqrsModule } from "@nestjs/cqrs";
+import { OrderStatus, TransactionLoggingService } from "../../../../../../../src/common";
+import { OrderPaidCommand } from "../../../../../../../src/listeners/substrate-listener/commands/orders";
+import { Test, TestingModule } from "@nestjs/testing";
+import { createMockOrder, mockBlockNumber, transactionLoggingServiceMockFactory } from "../../../../../mock";
+import { OrderPaidHandler } from "../../../../../../../src/listeners/substrate-listener/commands/orders/order-paid/order-paid.handler";
+
+describe("Order Paid Command Event", () => {
+  let orderPaidHandler: OrderPaidHandler;
+  let commandBus: CommandBus;
+
+  beforeEach(async () => {
+    const module: TestingModule = await Test.createTestingModule({
+      imports: [
+        CqrsModule
+      ],
```

Comment:
Never use `imports` in Testing environments, it's a bad practice. Provide the dependencies one by one in the `providers` section instead.